tests/test_data_streams.py

```diff
@@ -17,12 +17,12 @@
 
 
 def test_contrast_stream() -> None:
-    with open("tests/data/contrast-dump.pkls", "rb") as f:
+    with open("tests/data/contrast-dump.cbors", "rb") as f:
         while True:
             try:
-                frames = pickle.load(f)
+                frames = cbor2.load(f)
                 assert len(frames) == 1
-                data = pickle.loads(frames[0])
+                data = cbor2.loads(frames[0])
```

Comment:
Contrast has a pickle object as the first element of the zmq multipart message. We can't change Contrast to use cbor for now. So only the file container holding the zmq frames should be cbor, but the content itself is still a Contrast packet which uses cbor.